packages/core/utils/src/dml/helpers/entity-builder/define-property.ts

```diff
@@ -277,6 +279,31 @@ export function defineProperty(
     return
   }
 
+  /**
+   * Handling serial property separately to set the column type
+   */
+  if (field.dataType.name === "float") {
+    Property({
+      columnType: "numeric",
```

Comment:
q: shouldn't this be `real` to mirror Postgres?

Reply:
Double checked it and it seems that you can loose some precision with `real` and `double`. For example: If I run the following query where the `tax_rate_real` is a `REAL` column, then the updated value will be `0.102111116`.

```
UPDATE "public"."foo" SET "tax_rate_real" = 0.102111114;
```

Ofcourse, in our case we don't need these many decimal places, but will be nice to cover them for other use-cases with a more accurate data-type. WDYY?

Reply:
Hmm, I thought the argument for not using the `bigNumber` column here and introducing `float` was that we didn't care about precision for tax rates. If we want to solve for the scenario you describe, I would argue `bigNumber` is more appropriate.

Reply:
This looks good to me!

Your argument for using numeric makes sense–it offers higher precision for the rate of tax rates. That said, we introduced the `bigNumber` property to ensure consistency and leverage our existing utilities and that’s why I pushed back on adding `numeric as a new type. It’s not that high precision isn’t valid. I hope this rationale makes sense.